.gitignore

```diff
@@ -0,0 +1 @@
+/config/secrets.yml
```

Comment:
This worked, in that your secrets were kept safe. However, it made it impossible for me to test your app on my machine without writing extra code. If you had used dotenv gem, or alternatively set environment variables on your local machine, then I would have been able to figure out what was missing.